app-web/halloy/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=halloy
+PKGSEC=web
+PKGDEP="alsa-lib hicolor-icon-theme openssl glibc"
+BUILDDEP="cargo git llvm-20"
+PKGDES="An IRC client with GUI written in Rust"
```

Comment:
The "written in Rust" part is likely pointless information for the user. I looked up their README and found that they have a pretty good tile-based chat interface, so I would suggest:

Graphical IRC client with support for multiple tiled chat panels.